librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -19,63 +19,176 @@ commands:
             cname: cmd_debug_step_until
             summary: Step until <addr>
             args:
-            - name: addr
-              type: RZ_CMD_ARG_TYPE_RZNUM
+              - name: addr
```

Comment:
Are you sure these formatting changes are necessary? I recommend to recheck using `yamllint`.